repo/manifest/manifest_manager_test.go

```diff
@@ -293,7 +294,7 @@ func newManagerForTesting(ctx context.Context, t *testing.T, data blobtesting.Da
 
 	bm, err := content.NewManager(ctx, st, &content.FormattingOptions{
 		Hash:        "HMAC-SHA256-128",
```

Comment:
might as well change this for consistency
```suggestion
		Hash:        hashing.DefaultAlgorithm,
```